website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: The Anatomy of Unleash
+---
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+Unleash has many different components on different levels. This guide is here to give you a conceptual overview over how Unleash works. It covers the various components that exist within an Unleash system and how they interact with each other and with external applications. The diagrams are intended to help you understand the fundamental building blocks, such as [projects](../user_guide/projects.md), [environments](../user_guide/environments.md), [variants](../advanced/feature-toggle-variants.md) and, of course, [feature toggles](../reference/feature-toggles.mdx).
+
+The end of this guide presents a [short use case, explaining how you might configure Unleash](#use-case) to start working with feature toggles.
+
+
+## The global level
+
+Some things in Unleash are configured and defined on the global level. These options apply across the entire Unleash instance. The most important global configuration options for day-to-day operations are:
+
+- [API access tokens](../reference/api-tokens-and-client-keys.mdx)
+- [Projects](../user_guide/projects.md)
+- [Segments](../reference/segments.mdx)
+- [Strategy types](../user_guide/activation-strategies.md) (including [custom activation strategy types](../advanced/custom-activation-strategy.md))
+- [Tag types](../advanced/tags.md)
+- [Unleash context](../user_guide/unleash-context.md) fields (including [custom context fields](../user_guide/unleash-context.md#custom-context-fields))
+- Users, [user groups](../user_guide/rbac.md#user-groups) and [global roles](../user_guide/rbac.md)
+
+
+## Projects
+
+[**Projects**](../user_guide/projects.md) contain [feature toggles](../reference/feature-toggles.mdx) and their configurations, and a set of active [environments](../user_guide/environments.md).
+
+All Unleash instances must have at least one project at any given time. New instances get a project called “Default”.
+
+Pro and Enterprise customers can create, rename, and delete projects as they wish (as long as there is always **at least one project**). Open-source users, on the other hand, only get access to the default project.
```

Comment:
Very debatable suggestion here but I read this as a name since we did that in the last sentence (technically it's either a name or a descriptor so pick what makes your eyes happy)

```suggestion
Pro and Enterprise customers can create, rename, and delete projects as they wish (as long as there is always **at least one project**). Open-source users, on the other hand, only get access to the Default project.
```